wordpress-mobile/WordPress-iOS
# Fix existing UI tests

See https://github.com/wordpress-mobile/WordPress-iOS/issues/11013 for context.

This PR fixes the rest of our existing UI tests so they pass (previous fixes in https://github.com/wordpress-mobile/WordPress-iOS/pull/10987). It also adds some accessibility identifiers to the app, for use in the tests.

To test:

* Start the simulator.
* Run all of the tests for the WordPressUITests scheme and confirm they pass.
* If you get a build error related to the WPUITestCredentials, confirm your mobile secrets are up to date or replace the placeholders in `WPUITestCredentials.swift` with actual values.
